src/ds/d500/d500-mapping.cpp

```diff
@@ -0,0 +1,252 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#include "d500-mapping.h"
+
+#include <mutex>
+#include <chrono>
+#include <vector>
+#include <map>
+#include <iterator>
+#include <cstddef>
+
+#include "ds/ds-timestamp.h"
+#include "ds/ds-options.h"
+#include "stream.h"
+
+namespace librealsense
+{
+    
+    const std::map<uint32_t, rs2_format> mapping_fourcc_to_rs2_format = {
+        {rs_fourcc('G','R','E','Y'), RS2_FORMAT_OCCUP},
+        //point cloud - needed a standard format, with 32 bit per pixel:
+        // #define V4L2_PIX_FMT_ARGB32  v4l2_fourcc('B', 'A', '2', '4') /* 32  ARGB-8-8-8-8  */
+        {rs_fourcc('B','A','2','4'), RS2_FORMAT_XYZ32F}
```

Comment:
What is this format? temporary?

Reply:
Not temporary - this is the one written in the ICD